src/components/icons/AnimatedIcon.tsx

```diff
@@ -16,11 +20,21 @@ const AnimatedIcon = ({
   children,
   animationComplete,
 }: AnimatedIconType) => {
+  const isLightTheme = useSelector(state => theme(state) === 'Light')
+  const lottieColor = isLightTheme ? '#000000' : '#FFFFFF'
   const newSize = size * ICON_SCALING_FACTOR
   const color = style.fill || fill || token('colors.fg')
 
+  // Create a ref to the parent div
+  const divRef = useRef<HTMLDivElement | null>(null)
+
+  // Calculate dynamic color directly
+  const computedStyle = divRef.current ? getComputedStyle(divRef.current) : null
```

Comment:
```suggestion
  const computedStyle = divRef.current ? window.getComputedStyle(divRef.current) : null
```

This isn't too important, but nice for consistency. All other calls to this in the project explicitly use `window.getComputedStyle`.